### ISSUE TITLE
typescript

### ISSUE THREAD

**Issue by sgeisbacher** [pa-bru/graphql-cost-analysis#37]
are there any plans to switch from flow to typescript? i like your tool but its one of a few dependencies that neither has `@types/` -package nor has direct typescript-declaration in shipped npmjs.com-artifact. that prevents me from doing fully strict typescript-type-checking, cause such dependencies get implictAny type 😬

**Comment by sgeisbacher** [comment]
something like this: https://github.com/pa-bru/graphql-cost-analysis/compare/master...sgeisbacher:feature/intro-typescript?expand=1 (first untested draft)

**Comment by Ponjimon** [comment]
Some time ago, I created types for it but I didn't publish them as I don't know if they are sufficient. You might want to use them. 

https://gist.github.com/ponjimon/0c085f624291d7575dc5666b71768ce4

**Comment by daniyel** [comment]
@lookapanda Maybe you could create a PR for this?

**Comment by a7madgamal** [comment]
@Ponjimon the link gives 404, private gist?

**Comment by Ponjimon** [comment]
> @Ponjimon the link gives 404, private gist?

No, I just had a different username back then. I updated the link!
